src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js

```diff
@@ -124,6 +131,16 @@ describe('Autosuggest', () => {
             expect(wrapper.find('input')).toHaveLength(1);
             expect(wrapper.find('.Autosuggest__input--hidden')).toHaveLength(1);
         });
+        it('should show empty suggestion placeholder while results are empty and input has value', () => {
```

Comment:
I think description is confusing a bit. Maybe say `should show "no suggestions" placeholder only if input has value"